src/sentry/ingest/ingest_consumer.py

```diff
@@ -404,7 +256,7 @@ def process_individual_attachment(message: Message, project: Project) -> None:
 
 @trace_func(name="ingest_consumer.process_userreport")
 @metrics.wraps("ingest_consumer.process_userreport")
-def process_userreport(message: Message, project: Project) -> None:
+def process_userreport(message: IngestMessage, project: Project) -> None:
     start_time = to_datetime(message["start_time"])
     feedback = json.loads(message["payload"])
```

Comment:
Did we consider using rapidjson? IIRC we've seen some noticable improvements in Snuba and some other consumers when making this switch.

Reply:
Thanks, I've created a followup task in https://github.com/getsentry/team-processing/issues/46 to evaluate this. Once we've completed the move to multiprocessing, we can do another round of benchmarking. JSON parsing was relatively fast when compared with the database and cache access in the past.